test/lib/util.js

```diff
@@ -605,3 +605,130 @@ describe('Utils getLicenseLocations', () => {
     })
   })
 })
+
+describe('Utils buildSourceUrl', () => {
```

Comment:
While I was in here adding a test case for pypi packages, I figured I would add in some test cases for existing behavior as well.